### PR TITLE
Display reviewers on finding pages.

### PR DESCRIPTION
**Pull request by pedrohdjs**
**Description**

This PR adds a list of reviewers (assigned with the "request peer review" feature) to the UI in the finding pages.
I believe this would make for better finding visualization, and this is something that some users might miss, as noted in issue #10434's discussion.

**Test results**

I have tested (manually) all of the finding listing pages. Except when the findings are filtered by closed, all of the assigned reviewers should be displayed in the rightmost column.  

**Disclaimer**

Please, note that this is my first open source contribution PR ever, so, explaining stuff assuming I know very little about open source might be a good call. 

eli5

I'm open to any feedback and to implement any changes that might be necessary. Although I read the contribution guidelines, it's likely that I might have done some things wrong here, and I realize that this feature ideally should've gone through a pre-approval, but, since implementing it was fairly straightforward after I got (kind of :sweat_smile: ) used to the codebase and I wouldn't lose a lot of work if the PR is denied, I thought I'd give it a shot.

### CODE REVIEW

**Comment by pedrohdjs** [comment]
I mistakenly first created this pr targeting the master branch. That might've triggered [this failed check](https://github.com/DefectDojo/django-DefectDojo/pull/11165/checks?check_run_id=32296516687). I suggest re-running if possible (@mtesauro could you kindly do so, please?).

**Comment by devGregA** [comment]
Hi @pedrohdjs! Just wanted to pop in say thank you for the contribution and congrats on your first PR! I'm sure it will go smoothly. I have to leave the reviewing to the rest of the team because my python-foo isn't what it used to be, but just wanted to say hello and thank you.

**Comment by mtesauro** [comment]
@pedrohdjs Your PR is OK even if this test is failing:

image

That test is used to notify the core contributors that a PR is being done in an 'interesting' area of DefectDojo code from a non-core contributor. It's basically a flag to have us look a bit more closely at the PRs where that fails.

You're good to go with the tests as they are right now

**Review by Maffooch — APPROVED** [review]

**Review by hblankenship — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by grendel513 — APPROVED** [review]
Congrats on your first PR!
